proposals/load_ml_model.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "load_ml_model",
+    "summary": "Load a machine learning model",
+    "description": "...",
+    "categories": [
+        "machine learning",
+        "import"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "id",
+            "description": "The STAC Item to load the machine learning model from. The STAC Item must implement the ml-model extension. Such model can be trained with processes such as ``fit_random_forest()``.",
+            "schema": [
+                {
+                    "title": "URL",
+                    "type": "string",
+                    "format": "uri",
+                    "subtype": "uri",
+                    "pattern": "^https?://"
+                },
+                {
+                    "title": "Batch Job ID",
```

Comment:
How to specify the model to load (if there are multiple per job)?

Reply:
> Loading a model by batch job ID is possible only if a single model has been saved by the job. Otherwise, you have to load a specific model from a batch job by URL.